docs/gaia/gaiacli.md

```diff
@@ -87,15 +87,42 @@ Note that this is the Tendermint signing key, _not_ the operator key you will us
 We strongly recommend _NOT_ using the same passphrase for multiple keys. The Tendermint team and the Interchain Foundation will not be responsible for the loss of funds.
 :::
 
-#### Multisig public keys
+#### Generate multisig public keys
 
 You can generate and print a multisig public key by typing:
 
 ```bash
-gaiacli show --multisig-threshold K name1 name2 name3 [...]
+gaiacli keys add --multisig=name1,name2,name3[...] --multisig-threshold=K new_key_name
 ```
 
-`K` is the minimum weight, e.g. minimum number of private keys that must have signed the transactions that carry the generated public key.
+`K` is the minimum number of private keys that must have signed the
+transactions that carry the public key's address as signer.
+
+The `--multisig` flag must contain the name of public keys that will be combined into a
+public key that will be generated and stored as `new_key_name` in the local database. All names
+supplied through `--multisig` must already exist in the local database. The order in which the keys
+are supplied on the command line matters, i.e. the following commands generate two distinct
+multisig keys:
+
+```bash
+gaiacli keys add --multisig=foo,bar,baz --multisig-threshold=2 multisigaddress
+```
+
+```bash
+gaiacli keys add --multisig=baz,foo,bar --multisig-threshold=2 multisigaddress
+```
+
+Although keys and threshold parameter used in the above examples are the same, the commands
+generate two distinct keys due to the order in which the keys are specified.
+
+Multisig addresses can also be generated on-the-fly and printed through the which command:
```

Comment:
This needs a bit more explanation, or should be 🔪

Reply:
Done